src/frontend/components/Chat/index.tsx

```diff
@@ -13,7 +12,7 @@ interface ChatProps {
 
 const initConverse = converseMounter();
 
-export const Chat = ({ video: baseVideo, standalone }: ChatProps) => {
+export const Chat = ({ video: baseVideo }: ChatProps) => {
```

Comment:
Removes it from the props and everywhere it is used.

Reply:
And if you remove it you need to delete the route using it.

Reply:
I simplied my commit and now standalone is not removed anymore.